racunaljka.js

```diff
@@ -0,0 +1,230 @@
+/*=================================== brojevi==================================================*/
```

Comment:
The entire file is not well formatted, which makes it hard to read and follow. I suggest that for the entire project editing you install Sublime, since it is the most common simple editor with various plugins. One of those plugins is AutoFormatter, which will solve the formatting problem automatically. However, if you decide to stay Notepad++, just Google how to use formatter there.